Util/joi.ts

```diff
@@ -0,0 +1,47 @@
+import Joi from "joi";
+import { validationError } from "./custom.error";
+
+const idPattern = Joi.string().hex().length(24);
+
+const nameLength = Joi.object().keys({
+  name: Joi.string().required().max(10),
+});
```

Comment:
It doesn't need to be an object
```suggestion
const nameLength = Joi.string().required().max(10);
```